django/core/management/commands/runserver.py

```diff
@@ -65,6 +66,18 @@ def add_arguments(self, parser):
             action="store_true",
             help="Skip system checks.",
         )
+        parser.add_argument(
+            "--nostatic",
+            action="store_false",
+            dest="use_static_handler",
+            help="Tells Django to NOT automatically serve static files at STATIC_URL.",
+        )
+        parser.add_argument(
+            "--insecure",
+            action="store_true",
+            dest="insecure_serving",
+            help="Allows serving static files even if DEBUG is False.",
+        )
```

Comment:
Moving these options here is a change to the documented behavior that these options are only available when staticfiles is installed (see https://docs.djangoproject.com/en/5.1/ref/contrib/staticfiles/#cmdoption-runserver-nostatic)
Although this works, I don't think this is the right solution